packages/backend/docker-compose.yml

```diff
@@ -0,0 +1,14 @@
+version: '3.7'
```

Comment:
can you please coordinate with the assignees on #16 for the docker stuff

Reply:
will do. just didn't want to install locally.